cmd/newtemplates/browser.js

```diff
@@ -0,0 +1,60 @@
+import { browser } from "k6/browser";
+import http from "k6/http";
+import { sleep, check } from 'k6';
+
+const BASE_URL = __ENV.BASE_URL || "https://quickpizza.grafana.com";
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: "shared-iterations",
```

Comment:
I see in other examples we set duration/vus, should we set some "scope" here as well? 
Otherwise, does it bring any value to define the executor?

Reply:
👍🏻 Furthermore, my understanding is that in general users tend to run browser tests with 1 VU/1 iteration, at least when they get started. I know it's the default anyway, but as we're all addressing beginners with this command, it could be helpful to remove as much implicit as possible?

Reply:
Makes sense! I have added VUs/Iterations. 

> Otherwise, does it bring any value to define the executor?

It is needed to be able to run a browser test. You must use scenarios, and scenarios must have an executor.

Btw... we do this same thing in lots of other places; maybe it is something we should change?  cc @ankur22
e.g. https://grafana.com/docs/k6/latest/using-k6-browser/#a-simple-browser-test

Reply:
You mean adding the VU/iteration to all the other examples too? The situation with scenarios and browser tests isn't great tbh, and being forced to add an executor. I think we wanted to avoid making a very large options block, and the minimal is:

```js
export const options = {
  scenarios: {
    ui: {
      executor: 'shared-iterations',
      options: {
        browser: {
          type: 'chromium',
        },
      },
    },
  },
};
```

Adding iteration and vus adds a couple more lines to it.

If the general consensus is that it makes is clearer for new users then it's something to consider. CC @inancgumus

Reply:
I like being explicit in this k6 new command’s output and including VUs and iterations as outputted examples. Sure, we can also mention VUs and iterations in k6-docs here and there, but in general (all k6-docs, if that’s what we mean), I don’t believe we should add more clutter to that already crowded browser options block.